adi-diagnostic-report.desktop.in

```diff
@@ -3,6 +3,6 @@ Exec=@PREFIX@/bin/adi_diagnostic_report --gui
 Icon=utilities-system-monitor
 Terminal=false
 Type=Application
-Categories=System
+Categories=Science
```

Comment:
Isn't there a dedicated category like "Analog Devices tools" ?

Reply:
'Science' is the standard name, changed in kuiper to Analog Devices Tools